dev/deps/spark-deps-hadoop-3.2

```diff
@@ -102,17 +102,18 @@ httpclient-4.5.6.jar
 httpcore-4.4.10.jar
 istack-commons-runtime-3.0.8.jar
 ivy-2.4.0.jar
-jackson-annotations-2.9.10.jar
-jackson-core-2.9.10.jar
+jackson-annotations-2.10.0.jar
+jackson-core-2.10.0.jar
 jackson-core-asl-1.9.13.jar
-jackson-databind-2.9.10.jar
-jackson-dataformat-yaml-2.9.10.jar
+jackson-databind-2.10.0.jar
+jackson-dataformat-yaml-2.10.0.jar
 jackson-jaxrs-base-2.9.5.jar
 jackson-jaxrs-json-provider-2.9.5.jar
 jackson-mapper-asl-1.9.13.jar
-jackson-module-jaxb-annotations-2.9.10.jar
-jackson-module-paranamer-2.9.10.jar
-jackson-module-scala_2.12-2.9.10.jar
+jackson-module-jaxb-annotations-2.10.0.jar
+jackson-module-paranamer-2.10.0.jar
+jackson-module-scala_2.12-2.10.0.jar
+jakarta.activation-api-1.2.1.jar
```

Comment:
We'll need an entry in `LICENSE-binary` for this, under the Eclipse Dist License section. (We seem to already have a copy of the license though in `licenses-binary`).

Reply:
Done, thanks for pointing out